src/assets/examples/initiafy-action-bar/initiafy-action-bar.html

```diff
@@ -0,0 +1,3 @@
+<initiafy-action-bar>
+    Hello World!
```

Comment:
What about putting more data here to show the use of the Action Bar, or else it will just look like initiafy-card

Reply:
I understand where you are coming from. InitiafyCard and InitiafyActionBar are extremely similar. I wondered should there be more functionality in the action-bar, for example, I thought perhaps a search bar could be baked into the action bar (it is quite common to find there). What would you think of this? We could take this thread offline perhaps. 

What sort of content do you think would be good to put in here?